src/app/components/dashboard/dashboard.service.js

```diff
@@ -214,6 +214,8 @@
                             $rootScope.properties.hostname = 'Hostname not available.';
                             $log.error('Error fetching hostname.');
                         });
+              
+					PMAPIService.getMetricsMetadata(data);
```

Comment:
Yepp, closed the other PR. Up to you if it's easier to do all in the same PR. I'm Ok with having the metadata in $rootScope, since it might apply to other areas of the app, and also because the metrics are already in $rootScope. The only thing it would be prudent to do is to clean up the response and only store the parts we need from the metadata.